### PR TITLE
Pr 0.17

### LINKED ISSUE #79: Is this mod actively supported?
Most recent activity seems dated.  Interest caused by recent release of basegame v0.17.

### PR DESCRIPTION
**Pull request by kylewill0725**
Bare minimum update to 0.17. Closes #79

Tested:

* Checking and unchecking every textbox
* Toggling popup

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.4.302
+0.4.303
```
Comment (narc0tiq) [inline VERSION]: No need to bother with the version in future, I'll always make sure to take care of that.

**Comment by narc0tiq** [comment]
Thanks, this is awesome, I'll get it up in a few minutes!

**Comment by narc0tiq** [comment]
Thanks, man, I really appreciate the help with this and YARM; you're being awesome!

**Comment by kylewill0725** [comment]
Np, the server I'm on said that they would update to 0.17 when the mods are updated so I figured I would help speed it up :D

**Pull request merged**